lib/librato/rack/validating_queue.rb

```diff
@@ -5,6 +5,7 @@ class Rack
     # to work out the kinks
     #
     class ValidatingQueue < Librato::Metrics::Queue
+      DEFAULT_TAGS_LIMIT = 5
```

Comment:
Do we want to make this 4? I thought we had a scenario with 6 instrumentation-specific tags?